src/Microsoft.Extensions.FileProviders.Null/project.json

```diff
@@ -0,0 +1,34 @@
+﻿{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk",
+    "nowarn": [
+      "CS1591"
+    ],
+    "xmlDoc": true
+  },
+  "description": "Null file and directory providers for Microsoft.Extensions.FileProviders.",
```

Comment:
This just needs to be a class in the abstractions project. It has no dependencies.